lentils_tacos.md

```diff
@@ -0,0 +1,6 @@
+Using the same spices we can use cooked lentils instead of meat.
+*cooked lentils
+*different spices
+*mix and warm
```

Comment:
How to warm it @AdrianaMSA? 🤔

Kindly add more details.

Reply:
Ok I will make it better

Reply:
This reminds me that I was supposed to store the lentils and peas into the fridge 🙀

Reply:
Rush! 😱